### PR TITLE
Add new projectsUpdate query for improved pagination and memory and new projects feed

### PR DESCRIPTION
**Pull request by CarlosQ96**
Related to https://github.com/Giveth/Roadmap/issues/43

But related directly to the issue https://github.com/Giveth/impact-graph/issues/751#event-8334824283

### CODE REVIEW

**Comment by CarlosQ96** [comment]
@MoeNick New Query for frontend to implement:
```
query (
    $take: Int,
    $skip: Int
  ) {
    projectUpdates(
      take: $take,
      skip: $skip
    ) {
      projectUpdates {
        id
        title
        projectId
        userId
        content
        isMain
        totalReactions
        createdAt
        reaction {
          id
          userId
          reaction
          projectUpdateId
        }
        project {
          id
          slug
          totalReactions
        }
      }
      count
    }
  }
```

**Comment by CarlosQ96** [comment]
Default order by is createdAt Desc which fulfills the condition of the requirement

**Comment by CarlosQ96** [comment]
I feel like depending on the design I might have to update this query... so I am unsure if this is enough. Should we add more filters? or is this enough? @WhyldWanderer @MoeNick 

Right now this filters by createdAt. So the latest projectUpdate added  to giveth will show first. It's paginated.

**Comment by CarlosQ96** [comment]
do not merge until design is done.

File: src/resolvers/projectResolver.ts [hunk 1]
```diff
@@ -1519,6 +1534,55 @@ export class ProjectResolver {
...
+        FROM project_update as puu
+        WHERE puu."isMain" = false AND pu."projectId" = puu."projectId"
+        ORDER BY puu."createdAt" DESC
+        LIMIT 1
+      )
+      ORDER BY pu."createdAt" DESC
+      LIMIT ${take}
+      OFFSET ${skip}
+    `);
+
```
Comment (CarlosQ96) [inline src/resolvers/projectResolver.ts:1555]: I ran into this issues with distinctOn and joins, no amount of orderbys or groupbys to fix the error worked, and even if they did, it was returning not desired results. 

So I separated it to two queries for it to work.

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @CarlosQ96 
LGTM

**Comment by mohammadranjbarz** [comment]
> @MoeNick New Query for frontend to implement:
> 
> ```
> query (
>     $take: Int,
>     $skip: Int
>   ) {
>     projectUpdates(
>       take: $take,
>       skip: $skip
>     ) {
>       projectUpdates {
>         id
>         title
>         projectId
>         userId
>         content
>         isMain
>         totalReactions
>         createdAt
>         reaction {
>           id
>           userId
>           reaction
>           projectUpdateId
>         }
>         project {
>           id
>           slug
>           totalReactions
>         }
>       }
>       count
>     }
>   }
> ```

@MohammadPCh 
This PR has been merged, you can use this query

**Pull request merged**